examples/getting-started/wp-templates/single.tsx

```diff
@@ -21,8 +23,10 @@ export default function Single(): JSX.Element {
           { post && (
             <div>
               <div>
-                {/* eslint-disable-next-line react/no-danger */ }
-                <div dangerouslySetInnerHTML={ { __html: post.content ?? '' } } />
+                <HeadlessBlocks
+                  postContent={post.content ?? ''}
+                  blockComponents={blockComponents}
```

Comment:
Phil's [headless-blocks](https://npmjs.com/package/headless-blocks) parses the post content. When it finds block attributes, it looks in `blockComponents` to see if there's a React component for the block name.

If so, it renders that. Normal blocks like Core blocks are still rendered like before, as static HTML.